DEVELOPMENT.md

```diff
@@ -419,13 +419,10 @@ To perform a release run: `git checkout master && yarn release` and follow the
 instructions.
 
 Once the release PR branch is merged into master a build will be triggered on
-Buildkite, this will build Upstream for both Linux and macOS. When the build
-has completed you can download binaries for your platform [here][ar].
+Buildkite, this will build Upstream for both Linux and macOS (unsigned).
 
-Make sure to update the links to those binaries on the
-[radicle.xyz download page][rd] and rebuild/deploy the website.
-
-This is what a typical release looks like:
+<details>
+<summary>This is what a typical release looks like</summary>
```

Comment:
I was confused by this and thought it would show me an example of the release artifacts.
```suggestion
<summary>Commands to prepare a release</summary>
```